pliers/extractors/text.py

```diff
@@ -313,3 +334,52 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        verify_dependencies(['spacy'])
+        try: 
+            print(model)
+            model_na=model[:2]
```

Comment:
This is purely a stylistic thing that has no functional implications, but we try to adhere to the PEP8 style guide. Per PEP8, operators should have whitespace on both sides (i.e., `a = 2`, not `a=2`). This may seem pedantic, but it's good to get into the habit of following the conventions, because it makes code more readable (and most other Python codebases also follow PEP8, so it will benefit you later). Depending on your IDE/text editor, you can probably install a plugin that auto-formats the code for you.